x-pack/plugins/observability/public/pages/alerts/components/render_cell_value/render_cell_value.tsx

```diff
@@ -94,15 +96,7 @@ export const getRenderCellValue = ({
         const alert = parseAlert(observabilityRuleTypeRegistry)(dataFieldEs);
 
         return (
-          // NOTE: EuiLink automatically renders links using a <button>
-          // instead of an <a> when an `onClick` prop is provided, but this
-          // breaks text-truncation in `EuiDataGrid`, because (per the HTML
-          // spec), buttons are *always* rendered as `inline-block`, even if
-          // `display` is overridden. Passing an empty `href` prop forces
-          // `EuiLink` to render the link as an (inline) <a>, which enables
-          // text truncation, but requires overriding the linter warning below:
-          // eslint-disable-next-line @elastic/eui/href-or-on-click
-          <EuiLink href="" onClick={() => setFlyoutAlert && setFlyoutAlert(alert)}>
+          <EuiLink css={displayCss} onClick={() => setFlyoutAlert && setFlyoutAlert(alert)}>
```

Comment:
**note:** this element was rendered as an anchor tag to keep truncation on the content. This was causing a hard reload of the page on the Hosts View, and semantically was not representing an action.
Switching back to a button element and using the `display: contents` CSS properties allows keeping the same truncation behaviour.